<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Actividad 5. Algoritmos de Planificaci贸n - First Come First Served (FCFS)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="processForm">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lb_processName">
            <property name="text">
             <string>Nombre del proceso:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_processName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lb_processBurstTime">
            <property name="text">
             <string>Duraci贸n del proceso:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_processBurstTime"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="pb_process">
            <property name="text">
             <string>Agregar proceso</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Tabla de procesos</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QTableWidget" name="endedProcessTable"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cola de procesos</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="processTable"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Procesos terminados</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="processActual">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="0">
           <widget class="QPushButton" name="pb_pause">
            <property name="text">
             <string>Pausar</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pb_continue">
            <property name="text">
             <string>Continuar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lb_actualDuration">
            <property name="text">
             <string>Duraci贸n del proceso:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lb_actualName">
            <property name="text">
             <string>Nombre del proceso:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pb_end">
            <property name="text">
             <string>Finalizar</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Proceso en ejecuci贸n</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lb_actualState">
            <property name="text">
             <string>Estado del proceso:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLabel" name="lb_actualStateData">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLabel" name="lb_actualDurationData">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QLabel" name="lb_actualNameData">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lb_globalCounter">
            <property name="text">
             <string>Contador global:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_counterTime"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
